src/components/Match/MatchStory.jsx

```diff
@@ -109,6 +109,33 @@ const toSentence = (content) => {
   return result;
 };
 
+const pluralize = (count, object, pluralization_override=null) => {
```

Comment:
lint will probably want this in camelCase

Reply:
Oh, good call. Should I go ahead and camelCasify all of these snake_case variables? (e.g. pluralized_object, time_seconds, etc?)

Reply:
yes please.  `npm run lint` will tell you if you need to make any other lint fixes

Reply:
Awesome, thanks for the tip. Will make those changes.